cfgov/v1/wagtail_hooks.py

```diff
@@ -37,7 +37,7 @@
     video_player_test_cases,
 )
 from v1.util import util
-from v1.views.reports import DocumentsReportView, PageMetadataReportView
+from v1.views.reports import DocumentsReportView, ImagesReportView, PageMetadataReportView
```

Comment:
I think the linter will complain about a long line length here. You can have black auto-format things before commits to make the linter happy:

>Install pre-commit via pip or pipx, either globally or into your virtrualenv (recommended).
>
>`pip install -U pre-commit`
>
>Once it is installed, from the root of consumerfinance.gov repo, run the following command to install the pre-commit hooks
>
>`pre-commit install`
>
>This will allow pre-commit to run black, flake8, and isort, on commits where files being changed are relevant.